docs/02-app/01-building-your-application/02-data-fetching/02-server-actions-and-mutations.mdx

```diff
@@ -928,11 +929,12 @@ export default function Page() {
     ...
   }
 
-  return <button action={publish}>Publish</button>;
+  return <button onClick={publish}>Publish</button>;
```

Comment:
@samcx 
`onClick` pass `MouseEvent` not `FormData`. `formAction` was correct

Reply:
@A7med3bdulBaset Thanks for the correction, will update!